### DISCUSSION TITLE
Flie not showing up via editor in OSU!lazar

### DISCUSSION THREAD

**Discussion by SkyyyXD** [ppy/osu#23100] [Q&A]
My Mp3 file which contains music isn't showing up in the editor file. Can someone please help me with this.
Image of the problem:
![Screenshot_20230402-213043](https://user-images.githubusercontent.com/129709059/229412264-de3a727f-3b4a-4a10-bdd2-d2a414119d91.png)

**Comment by peppy** [comment]
Can you show that folder in another file explorer app?

**Reply by SkyyyXD** [reply]
Sure thing! 
![Screenshot_20230402-212712](https://user-images.githubusercontent.com/129709059/229411561-a5bef29f-f315-4f2e-9a73-af55c4973222.png)
This is in the download folder. Which in the photo that I provided was in the music folder sorry. But, when I check the download folder it doesn't show. Here is the other image for the OSU!lazar editor folder showing nothing. 
![Screenshot_20230402-213043](https://user-images.githubusercontent.com/129709059/229411948-248cfb25-351f-4bac-8436-1f5b0de43145.png)

**Comment by peppy** [comment]
And you definitely clicked "audio track" and not accidentally "background"?

**Reply by SkyyyXD** [reply]
I clicked on audio track and It continues to not show my mp3 file.

**Reply by SkyyyXD** [reply]
Figured it out. Thank you so much!

**Reply by peppy** [reply]
What was the issue?

**Reply by SkyyyXD** [reply]
Sorry for not responding, I just gave the app permission to access to my flies.
